Assignment - Mathematical Exponentiation

chinmayjain08 opened the pull request:
Solved a mathematical expression of form (B^N % M) allowing large values for B and N.
Used the property of modulo(%) : (a*b) % m = [(a % m) * (b % m)] % m , to achieve the desired result.
![image](https://github.com/user-attachments/assets/18f9d87a-f16e-4bfb-be9a-ae637deed643)

chinmayjain08 committed:
assignment-MathematicalProblem-submit

chinmayjain08 committed:
logic-for-mod-operation-property-added

chandan2853 commented on assignmentModuloExponentiation.c:
```diff
@@ -0,0 +1,30 @@
+#include<stdio.h>
```
Why there is no input or error handling done? and naming convention

chandan2853 replied:
giving incorrect output if i give base negative

chinmayjain08 replied:
input validation is handled and naming is done in a more semantic manner

chinmayjain08 committed:
input-validation handled

input validation done, ensuring a valid positive input for all the parameters required from user.
